### PR TITLE
Porting SettingsControls

### PR DESCRIPTION
**Pull request by niels9001**
@michael-hawker @Arlodotexe can we merge this as is and we add tests later? that would unblock me from using it for https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/110/

### CODE REVIEW

File: components/SettingsControls/src/AdditionalAssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+// These `InternalsVisibleTo` calls are intended to make it easier for
+// for any internal code to be testable in all the different test projects
+// used with the Labs infrastructure.
```
Comment (Arlodotexe) [inline components/SettingsControls/src/AdditionalAssemblyInfo.cs:9]:
```suggestion
// used with the Toolkit infrastructure.
```

**Review by Arlodotexe — APPROVED** [review]

**Comment by michael-hawker** [comment]
@Arlodotexe even weird CI issues here...?

File: components/SettingsControls/samples/Assets/SettingsCard.png
Comment (michael-hawker) [inline components/SettingsControls/samples/Assets/SettingsCard.png:1]: If we don't have an icon for this one yet, can we at least get a better default to start using? 😋

File: components/SettingsControls/src/Dependencies.props [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  <!-- WinUI 3 / WinAppSdk -->
+  <ItemGroup Condition="'$(IsWinAppSdk)' == 'true'">
+    <PackageReference Include="CommunityToolkit.WinUI.Triggers" Version="8.0.0-beta.1"/>
+  </ItemGroup>
+
+  <!-- WinUI 3 / Uno -->
+  <ItemGroup Condition="'$(IsUno)' == 'true' AND '$(WinUIMajorVersion)' == '3'">
+    <PackageReference Include="CommunityToolkit.WinUI.Triggers" Version="8.0.0-beta.1"/>
+  </ItemGroup>
```
Comment (michael-hawker) [inline components/SettingsControls/src/Dependencies.props:30]: Ah, this is the CI issue, you don't need a package reference anymore, just need to add a project reference in the csproj to the Triggers project.

(Don't add a thing in the root props file like was done elsewhere, just use the path in the project reference in the csproj. Though we had an example somewhere, but can't find it atm)

**Review by michael-hawker — CHANGES_REQUESTED** [review]
Couple of small things to fix.

No code changes, just the namespace right?

**Review by michael-hawker — COMMENTED** [review]
🦙❤

**Pull request merged**